overview.html

```diff
@@ -0,0 +1,452 @@
+<html>
+<!--
+This overview.html provides Javadoc overview for the application.
+The content from the body section is added to the Javadoc overview page.
+There is no need to add a head/title as that is provided by Javadoc.
+-->
+<body>
+
+<P>
+    This is the official Cloudant library for java.
+</P>
+
+<OL>
+    <LI><a href="#Initialization">Initialization</a>
+        <OL>
+            <LI><a href="#Cloudant service example">Cloudant service example</a></LI>
+            <LI><a href="#Cloudant Local example">Cloudant Local example</a></LI>
```

Comment:
Good point, it appears to be called "Cloudant Local" colloquially on the official site, so I have used the full name in the first occurrence in the "Compatibility" list in 9a7715e and left as Cloudant Local elsewhere.